src/multitrack.ts

```diff
@@ -131,9 +131,15 @@ class MultiTrack extends EventEmitter<MultitrackEvents> {
   }
 
   private initAudio(track: TrackOptions): Promise<HTMLAudioElement> {
-    const audio = new Audio()
+    let audio = new Audio()
```

Comment:
I would move the audio creation to the `if`. I.e.

```js
    let audio: AudioElement
    if (track.url) {
       audio = new Audio()
       audio.crossOrigin = 'anonymous'
       audio.src = track.url
     } else if (track.options?.media) {
       audio = track.options?.media
     }
```

Reply:
If I don't initialize Audio() early on, an error message appears, saying Variable 'audio' is used before being assigned. I believe that audio should still be created even if the user doesn't provide either the url parameter or the options.media parameter.

Alternatively, other approaches come to mind, such as adding an else statement or using type assertion. However, early initialization of audio seems like the better option to me. If I'm misunderstanding something or if there's a better approach, please let me know.

error message
<img width="847" alt="image" src="https://github.com/katspaugh/wavesurfer-multitrack/assets/47740690/a93c404f-bbb9-4751-9307-558dcfb06a68">

Reply:
Correct, it should create an audio element without src if neither url nor media are provided. So yeah, an else clause at the end would work.